recipe/migrations/libprotobuf215.yaml

```
@@ -0,0 +1,9 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+# 3.20 is the last protobuf migration in flight
+libprotobuf:
+- '3.20'
+- '21.5'
```

```suggestion
- '3.21.5'
```
It seems we're going to have to continually add the major version back here.